src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -668,6 +671,13 @@ class Address {
     }
     return false;
   }
+
+  #define create_imm_offset(klass, field_offset_func) \
```

Comment:
I would avoid "klass" as name, since it is usually implied to mean an object of class `Klass`.